services-directions-models/src/test/resources/directions-with-closure.json

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
should we rename the file to "fit in" `directions_v5_closure.json`

![Screen Shot 2021-03-30 at 1 35 18 PM](https://user-images.githubusercontent.com/3021882/113053123-c7cc3600-915c-11eb-99fb-f68f288bad8a.png)

Reply:
renamed.